distributions/Registration/build.gradle

```diff
@@ -41,8 +41,8 @@ project.task(
         type: ModuleDistribution,
         {ModuleDistribution dist ->
             dist.subDirName='Registration'
-            dist.versionPrefix="Registration-1.38"
-            dist.extraFileIdentifier='-Registration-1.38'
+            dist.versionPrefix="Registration"
```

Comment:
We should just remove the `versionPrefix`. The gradle plugin doesn't support it anymore.

Reply:
Removed `versionPrefix` and `description` (Susan confirmed it's not used)